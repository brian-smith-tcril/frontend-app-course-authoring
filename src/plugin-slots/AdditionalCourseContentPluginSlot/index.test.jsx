import { shallow } from '@edx/react-unit-test-utils';
import { AdditionalCourseContentPluginSlot } from '.';

jest.mock('@openedx/frontend-plugin-framework', () => ({
  PluginSlot: 'PluginSlot',
}));

describe('AdditionalCourseContentPluginSlot', () => {
  beforeEach(() => jest.resetAllMocks());

  it('renders', () => {
    const wrapper = shallow(<AdditionalCourseContentPluginSlot />);
    expect(wrapper.snapshot).toMatchSnapshot();
  });
});
