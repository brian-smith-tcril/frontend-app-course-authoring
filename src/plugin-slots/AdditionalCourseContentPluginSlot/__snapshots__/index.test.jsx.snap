// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdditionalCourseContentPluginSlot renders 1`] = `
<PluginSlot
  id="org.openedx.frontend.authoring.additional_course_content_plugin.v1"
  idAliases={
    [
      "additional_course_content_plugin",
    ]
  }
/>
`;
